pkg/types/gcp/validation/machinepool.go

```diff
@@ -64,7 +64,7 @@ func ValidateServiceAccount(platform *gcp.Platform, p *types.MachinePool, fldPat
 func ValidateMasterDiskType(p *types.MachinePool, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 	if p.Name == "master" && p.Platform.GCP.OSDisk.DiskType != "" {
-		diskTypes := sets.NewString("pd-ssd")
+		diskTypes := sets.NewString("pd-ssd", "pd-balanced", "hyperdisk-balanced")
```

Comment:
As mentioned above, let's not duplicate this.

Reply:
If we are validating the disk type during install-config validation, do we need to do it here as well?